### PR TITLE
LPS-183908| Test Fix | Master

### PR DESCRIPTION
**Pull request by davisaints**
**JIRA Ticket: [LPS-183908](https://issues.liferay.com/browse/LPS-183908)**

### CODE REVIEW

**Comment by davisaints** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/IFrame.path [hunk 1]
```diff
@@ -62,7 +62,7 @@
 </tr>
 <tr>
 	<td>DIALOG_BOX</td>
-	<td>//div[contains(@class,'modal-header')]/h3</td>
+	<td>//div[contains(@class,'modal-header')]</td>
```
Comment (davisaints) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/IFrame.path:65]: This change fixes the path being used in the teardown of this file, also fixing the tests reported in [this]([LPS-184151](https://issues.liferay.com/browse/LPS-184151)) other ticket.

**Comment by davisaints** [comment]
ci:test:data-engine

**Comment by davisaints** [comment]
No related failures.

cc @Luizcarlosqueiroz

**Comment by davisaints** [comment]
ci:test:sf

**Review by Luizcarlosqueiroz — APPROVED** [review]
Great job here! :clap:

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/IFrame.path [hunk 2]
```diff
@@ -62,7 +62,7 @@
 </tr>
 <tr>
 	<td>DIALOG_BOX</td>
-	<td>//div[contains(@class,'modal-header')]/h3</td>
+	<td>//div[contains(@class,'modal-title')]</td>
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/IFrame.path:65]: @davisaints do you think we can use some existing path/s in Modal.path so we don't create paths that could be duplicates

**Review by patriciajeanperez — COMMENTED** [review]
Nice work, @davisaints! Thank you for combining all related tickets. Other than potentially checking if we can use existing locators in Modal.path, this PR looks good to me.

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/IFrame.path (hunk 2 above)
Comment (davisaints) [reply portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/IFrame.path]: This change is no longer necessary as it has already been made in another pr.

**Comment by davisaints** [comment]
ci:test:sf

**Pull request closed without merging**